azurerm/resource_arm_monitor_action_group_test.go

```diff
@@ -440,41 +954,319 @@ resource "azurerm_monitor_action_group" "test" {
+
+		webhook_receiver {
+			name        = "callmyapiaswell"
+			service_uri = "http://example.com/alert"
+		}
+
+		webhook_receiver {
+			name        = "callmybackupapi"
+			service_uri = "https://backup.example.com/warning"
+		}
+
+		automation_runbook_receiver {
+			name = "action_name_1"
+			automation_account_id = "${azurerm_automation_account.test.id}"
+    	runbook_name = "${azurerm_automation_runbook.test.name}"
+			webhook_resource_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/rg-runbooks/providers/microsoft.automation/automationaccounts/aaa001/webhooks/webhook_alert"
+			is_global_runbook = true
+			service_uri = "https://s13events.azure-automation.net/webhooks?token=randomtoken"
+		}
+
+		voice_receiver {
+			name         = "oncallmsg"
+			country_code = "1"
+			phone_number = "1231231234"
+		}
+
+		logic_app_receiver {
+			name = "logicappaction"
+			resource_id = "${azurerm_logic_app_workflow.test.id}"
+			callback_url = "http://test-host:100/workflows/fb9c8d79b15f41ce9b12861862f43546/versions/08587100027316071865/triggers/manualTrigger/paths/invoke?api-version=2015-08-01-preview&sp=%%2Fversions%%2F08587100027316071865%%2Ftriggers%%2FmanualTrigger%%2Frun&sv=1.0&sig=IxEQ_ygZf6WNEQCbjV0Vs6p6Y4DyNEJVAa86U5B4xhk"
+		}
+
+		azure_function_receiver {
+			name = "funcaction"
+			function_app_resource_id = "${azurerm_function_app.test.id}"
+			function_name = "myfunc"
+			http_trigger_url = "https://example.com/trigger"
+		}
+
+	}
+
+	resource "azurerm_automation_account" "test" {
+		name                = "acctestAA-%d"
+		location            = "${azurerm_resource_group.test.location}"
+		resource_group_name = "${azurerm_resource_group.test.name}"
+
+		sku {
+		  name = "Basic"
+		}
+	}
+
+	resource "azurerm_automation_runbook" "test" {
+		name                = "Get-AzureVMTutorial"
+		location            = "${azurerm_resource_group.test.location}"
+		resource_group_name = "${azurerm_resource_group.test.name}"
+		account_name        = "${azurerm_automation_account.test.name}"
+		log_verbose         = "true"
+		log_progress        = "true"
+		description         = "This is an test runbook"
+		runbook_type        = "PowerShellWorkflow"
+
+		publish_content_link {
+		  uri = "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-automation-runbook-getvms/Runbooks/Get-AzureVMTutorial.ps1"
+		}
+	}
+
+	resource "azurerm_logic_app_workflow" "test" {
+		name                = "acctestLA-%d"
+		location            = "${azurerm_resource_group.test.location}"
+		resource_group_name = "${azurerm_resource_group.test.name}"
+	}
+
+	resource "azurerm_logic_app_trigger_http_request" "test" {
+		name         = "some-http-trigger"
+		logic_app_id = "${azurerm_logic_app_workflow.test.id}"
+
+	schema = <<SCHEMA
+	{
+		"type": "object",
+		"properties": {
+			"hello": {
+				"type": "string"
+			}
+		}
+	}
+	SCHEMA
+	}
+
+	resource "azurerm_storage_account" "test" {
+		name                     = "acctestSA-%d"
```

Comment:
(as above) could we make this:

```suggestion
		name                     = "acctestsa%s"
```